domains/ramkumar.json

```
@@ -4,6 +4,7 @@
         "email": "ramkumar213supm@gmail.com"
     },
     "records": {
-        "CNAME": "jerrydev210.github.io"
+        "CNAME": "jerrydev210.github.io",
+        "TXT": "google-site-verification=7k6hDebRVVhSMPfTDftyAt-emDyC_FZNwkkUrs8GZP4"
     }
 }
```

CNAME cant be combine with other records unless proxied

```suggestion
},
"proxied": true
```

---

Thanks for making the changes for me